docs/ruleset_functions/expected_system_type_from_Table_G3_1_1.md

```diff
@@ -0,0 +1,117 @@
+# expected_system_type_from_Table_G3_1_1
+**Schema Version:** 0.0.22  
+
+**Description:** returns the expected system type based on the BAT, the number of floors of the building and the area of the BAT, and climate zone
+
+**Inputs:**
+- **building_area_type** - ("RESIDENTIAL", "PUBLIC_ASSEMBLY", "RETAIL", "HOSPITAL", "OTHER_NON_RESIDENTIAL", "HEATED-ONLY_STORAGE")
+- **number_of_floors** - this is the number of floors in the building integer greater than 0
+- **area_of_BAT** - this is the total area of the building area type
+
+**Returns:**  
+- **result**: a dict indicating the expected system type from table G3_1_1 ("SYS-1", "SYS-2", etc) and a string that indicates how the system was chosen, for example: {"EXPECTED_SYSTEM_TYPE": "SYS-4", "SYSTEM_ORIGIN": "PUBLIC_ASSEMBLY CZ_0_to_3a < 120,000 ft2"} - does not take into account G3.1.1 b-g
+ 
+**Function Call:**
+- **is_CZ_0_to_3a**
+
+## Logic:
+- create area_of_BAT_string, which will be appended to the details_of_system_selection string: `area_of_BAT_string = ""`
+- create number_of_floors_string, which will be appended to the details_of_system_selection string: `number_of_floors_string = ""`
+
+- get the climate zone category using the functions is_CZ_0_to_3a: `if is_CZ_0_to_3a():`
+	- set climate_zone_category to CZ_0_to_3a: `climate_zone_category = " CZ_0_to_3a"
+
+- else (the climate zone is 3b, 3c or 4 to 8): `else:`
+	- set climate_zone_category to CZ_3b_3c_or_4_to_8: `climate_zone_category = " CZ_3b_3c_or_4_to_8"
+
+- check if the building area type is residential: `if building_area_type == "RESIDENTIAL":`
+	- check if the climate zone is CZ_0_to_3a: `if is_CZ_0_to_3a():`
+		- the expected system is System 1: `system_type = "SYS-2"`
+	- otherwise, it's  CZ_3b_3c_or_4_to_8: `else:`
+		- the expected system is System 2: `system_type = "SYS-1"
+
```

Comment:
again, text needs to be revised

Reply:
should we also check that the area is >20,000 ft2? the exxpected system will not be 1 or 2 if area is <20,000 ft2, correct?

Reply:
@supriyagoel - are you referring to G3.1.1.b? (Use additional system types for non-predominant conditions (i.e., residential/nonresidential) if those conditions apply to more than 20,000 ft2 of conditioned floor area.)

The purpose of this function is only to select the appropriate system type based on the given `building_area_type`, `number_of_floors`, `area_of_BAT` and climate zone - it does HVAC system selection from table G3.1.1-3 only on these variables.

there are subsequent functions that over-write the system selection and target string if one of the sub-items in G3.1.1 apply.